packages/tests/server/adapters/fastify.test.ts

```diff
@@ -199,9 +199,7 @@ describe('anonymous user', () => {
     await app.start();
   });
 
-  afterEach(async () => {
-    await app.stop();
-  });
```

Comment:
https://typescript-eslint.io/rules/require-await

`app.stop()` doesn't create a `Promise` according to its types. That should indicate there's no need to `await` it.

If there _is_ a need to `await` it, then its return type should indicate it's returning a `Promise`.